src/components/UIColumn.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { Album } from '../interfaces/album';
+import { Song } from '../interfaces/song';
+
+import { AlbumList } from './AlbumList';
+import { IconTextField } from './IconTextField';
+
+interface Props {
+  albums: Array<Album>;
+  songs: Array<Song>;
+  onSongClick: Function;
+}
+
+interface State {
+  searchQuery: string;
+}
+
+export class UIColumn extends React.Component<Props, State> {

```

Comment:
this seems to be doing a lot of very specific things for a component named `UIColumn`